powertools-core/src/main/resources/version.properties

```diff
@@ -0,0 +1 @@
+powertools.version=${project.version}
```

Comment:
Will this be filled by the build as it exists currently or do we need to do something special?

Reply:
see my comment [here](https://github.com/aws-powertools/powertools-lambda-java/pull/1306#discussion_r1270759687)